src/hardware/dma.cpp

```diff
@@ -264,22 +264,26 @@ Bitu DmaController::ReadControllerReg(Bitu reg,Bitu /*len*/) {
 	return 0xffffffff;
 }
 
-DmaChannel::DmaChannel(Bit8u num, bool dma16) {
-	masked = true;
-	callback = NULL;
-	if(num == 4) return;
-	channum = num;
```

Comment:
If `num` is 4, then `channum` is left undefined?

Reply:
Great find, thank you!

This `if` statement generally looks fishy, but I don't want to jump into another rabbit hole just yet ;)